addapted all path to gentoo system
typo fixed

Joerg Bornkessel <hd_brummy@gentoo.org> 04 Jan 2007
diff -Naur dvdconvert-1.0.2.orig/dvdconvert.c dvdconvert-1.0.2/dvdconvert.c
--- dvdconvert-1.0.2.orig/dvdconvert.c	2007-01-05 20:09:55.308636742 +0100
+++ dvdconvert-1.0.2/dvdconvert.c	2007-01-05 20:10:06.461272295 +0100
@@ -74,12 +74,12 @@
    char *p1, *p2;
 
    if( cfgFile == NULL )
-      strcpy(configFile,AddDirectory( cPlugin::ConfigDirectory(), CFG_FILE ));
+      strcpy(configFile,AddDirectory( cPlugin::ConfigDirectory("../../../usr/share/vdr/dvdconvert"), CFG_FILE ));
    else
       strcpy(configFile,cfgFile);
 
    if( scrFile == NULL )
-      strcpy(scriptFile,AddDirectory( cPlugin::ConfigDirectory(), SCRIPT_FILE ));
+      strcpy(scriptFile,AddDirectory( cPlugin::ConfigDirectory("../../../usr/share/vdr/dvdconvert/bin"), SCRIPT_FILE ));
    else
       strcpy(scriptFile,scrFile);
 
diff -Naur dvdconvert-1.0.2.orig/scripts/dvd2dvd-1.0.2.sh dvdconvert-1.0.2/scripts/dvd2dvd-1.0.2.sh
--- dvdconvert-1.0.2.orig/scripts/dvd2dvd-1.0.2.sh	2007-01-05 20:09:55.300636285 +0100
+++ dvdconvert-1.0.2/scripts/dvd2dvd-1.0.2.sh	2007-01-05 20:10:24.358292186 +0100
@@ -33,7 +33,7 @@
 WORKDIR=/video0/vdr/tmp/dvd2dvd	# working directory of dvd2dvd
 DVD_DEVICE=/dev/cdrom				# dvd device
 
-DVD2DVD_CONF=/video0/vdr/conf/plugins/dvd2dvd.conf
+DVD2DVD_CONF=/etc/vdr/plugins/dvdconvert/dvdconvert
 
 					# this file overrides some
 					# internal settings and
@@ -111,7 +111,7 @@
 DVDRECORD_OPT_2="-dao driveropts=burnfree"
 DVDFORMAT_OPT="-Z"
 
-LOGDIR=/var/log/vdr
+LOGDIR=/var/log/dvdconvert
 LOGFILE=$LOGDIR/dvd2dvd.log
 LOCKFILE=$LOGDIR/dvd2dvd.LOCK
 
diff -Naur dvdconvert-1.0.2.orig/scripts/dvd2vdr-1.0.1.sh dvdconvert-1.0.2/scripts/dvd2vdr-1.0.1.sh
--- dvdconvert-1.0.2.orig/scripts/dvd2vdr-1.0.1.sh	2007-01-05 20:09:55.284635374 +0100
+++ dvdconvert-1.0.2/scripts/dvd2vdr-1.0.1.sh	2007-01-05 20:10:06.529276171 +0100
@@ -32,7 +32,7 @@
 WORKDIR=/video0/vdr/tmp/dvd2vdr	# working directory of dvd2vdr
 DVD_DEVICE=/dev/cdrom			# dvd device
 
-DVD2VDR_CONF=/video0/vdr/conf/plugins/dvd2vdr.conf
+DVD2VDR_CONF=/etc/vdr/plugins/dvdconvert/dvdconvert
 
 					# this file overrides some
 					# internal settings and
@@ -97,7 +97,7 @@
 GENINDEX_BIN=genindex
 GENINDEX_MJPEGFIX_BIN=genindex-mjpegfix
 
-LOGDIR=/var/log/vdr
+LOGDIR=/var/log/dvdconvert
 LOGFILE=$LOGDIR/dvd2vdr.log
 LOCKFILE=$LOGDIR/dvd2vdr.LOCK
 
@@ -654,7 +654,7 @@
 		cat ${TEMPDIR}/vobcopy_*.log >> $LOGFILE
 		if [ $TITLENUM = "AUTO" ] ; then	
 			TITLENUM=`cat ${TEMPDIR}/vobcopy_*.log | \
-			grep 'Using Title' | head -n 1 | awk '{print $4}'` && \
+			grep 'Using Title' | head -n 1 | awk '{print $4}'`
 		fi
 		;;
 	esac
diff -Naur dvdconvert-1.0.2.orig/scripts/dvdconvert.conf dvdconvert-1.0.2/scripts/dvdconvert.conf
--- dvdconvert-1.0.2.orig/scripts/dvdconvert.conf	2007-01-05 20:09:55.292635831 +0100
+++ dvdconvert-1.0.2/scripts/dvdconvert.conf	2007-01-05 20:10:06.537276626 +0100
@@ -4,39 +4,39 @@
 #
 :dvd2vdr
 --- Einstellungen_dvd2vdr ---
-/usr/local/bin/start_dvd2vdr.sh:ACTION:0:L:1:all,copy,demux,encode,mplex,split,move:Actions:
-/usr/local/bin/start_dvd2vdr.sh:LANGUAGE:0:L:1:de,en:Language:
-/usr/local/bin/start_dvd2vdr.sh:OSDINFO:1:B:0:Nein,Ja:OSD-Info:
-/usr/local/bin/start_dvd2vdr.sh:TITLE: :A:15:abcdefghijklmnopqrstuvwxyz0123456789-_ABCDEFGHIJKLMNOPQRSTUVWXYZ:Title (optional):
-/usr/local/bin/start_dvd2vdr.sh:TITLENUM: :A:3:0,99,MAX,AUTO:Specific title (00-99) or (MAX = max. Frames) or (AUTO = vobcopy-test)(optional):
-/usr/local/bin/start_dvd2vdr.sh:ACTION_DEMUX:0:L:0:tcextract,projectX:Demuxer:
-/usr/local/bin/start_dvd2vdr.sh:WITHOUT_X:0:B:1:Nein,Ja:Start Xvfb (optional):
-/usr/local/bin/start_dvd2vdr.sh:A52DEC_GAIN: :I:2:-96,96:Audio gain (-96.0 to +96.0)(optional):
-/usr/local/bin/start_dvd2vdr.sh:ACTION_ENCODE:0:L:0:toolame,mp2enc:Encoder:
-/usr/local/bin/start_dvd2vdr.sh:ACTION_MPLEX:0:L:0:tcmplex,tcmplex-panteltje:Remuxer:
-/usr/local/bin/start_dvd2vdr.sh:ACTION_GENINDEX:0:L:0:genindex-mjpegfix,genindex:Write index.vdr:
-/usr/local/bin/start_dvd2vdr.sh:REMOVE:0:B:1:Nein,Ja:Remove tmp files:
-/usr/local/bin/start_dvd2vdr.sh:CLEAN:0:B:1:Nein,Ja:Clean dirs:
-/usr/local/bin/start_dvd2vdr.sh:VERBOSE:1:B:1:Nein,Ja:Turns verbosity on:
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:ACTION:0:L:1:all,copy,demux,encode,mplex,split,move:Actions:
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:LANGUAGE:0:L:1:de,en:Language:
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:OSDINFO:1:B:0:Nein,Ja:OSD-Info:
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:TITLE: :A:15:abcdefghijklmnopqrstuvwxyz0123456789-_ABCDEFGHIJKLMNOPQRSTUVWXYZ:Title (optional):
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:TITLENUM: :A:3:0,99,MAX,AUTO:Specific title (00-99) or (MAX = max. Frames) or (AUTO = vobcopy-test)(optional):
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:ACTION_DEMUX:0:L:0:tcextract,projectX:Demuxer:
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:WITHOUT_X:0:B:1:Nein,Ja:Start Xvfb (optional):
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:A52DEC_GAIN: :I:2:-96,96:Audio gain (-96.0 to +96.0)(optional):
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:ACTION_ENCODE:0:L:0:toolame,mp2enc:Encoder:
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:ACTION_MPLEX:0:L:0:tcmplex-panteltje:Remuxer:
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:ACTION_GENINDEX:0:L:0:genindex:Write index.vdr:
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:REMOVE:0:B:1:Nein,Ja:Remove tmp files:
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:CLEAN:0:B:1:Nein,Ja:Clean dirs:
+/usr/share/vdr/dvdconvert/bin/start_dvd2vdr.sh:VERBOSE:1:B:1:Nein,Ja:Turns verbosity on:
 :
 ---
 #
 ---
 :dvd2dvd
 --- Einstellungen_dvd2dvd ---
-/usr/local/bin/start_dvd2dvd.sh:ACTION1:0:L:1:all,copy,demux,encode,reqant,mplex,dvdauthor,iso,move:Actions:
-/usr/local/bin/start_dvd2dvd.sh:LANGUAGE1:0:L:1:de,en:Language:
-/usr/local/bin/start_dvd2dvd.sh:OSDINFO1:1:B:0:Nein,Ja:OSD-Info:
-/usr/local/bin/start_dvd2dvd.sh:TITLE1: :A:15:abcdefghijklmnopqrstuvwxyz0123456789-_ABCDEFGHIJKLMNOPQRSTUVWXYZ:Title (optional):
-/usr/local/bin/start_dvd2dvd.sh:TITLENUM1: :A:3:0,99,MAX,AUTO:Specific title (00-99) or (MAX = max. Frames) or (AUTO = vobcopy-test)(optional):
-/usr/local/bin/start_dvd2dvd.sh:ACTION_DEMUX1:0:L:0:tcextract,projectX:Demuxer:
-/usr/local/bin/start_dvd2dvd.sh:WITHOUT_X1:0:B:1:Nein,Ja:Start Xvfb (optional):
-/usr/local/bin/start_dvd2dvd.sh:A52DEC_GAIN1: :I:2:-96,96:Audio gain (-96.0 to +96.0)(optional):
-/usr/local/bin/start_dvd2dvd.sh:ACTION_ENCODE1:0:L:0:toolame,mp2enc:Encoder:
-/usr/local/bin/start_dvd2dvd.sh:ACTION_MPLEX1:0:L:0:tcmplex,tcmplex-panteltje:Remuxer:
-/usr/local/bin/start_dvd2dvd.sh:BURN1:1:B:1:Nein,Ja:Burn DVD:
-/usr/local/bin/start_dvd2dvd.sh:RW_FORMAT1:0:B:1:Nein,Ja:Format DVD-RW:
-/usr/local/bin/start_dvd2dvd.sh:REMOVE_ISO1:0:B:1:Nein,Ja:Remove this iso file:
-/usr/local/bin/start_dvd2dvd.sh:REMOVE1:0:B:1:Nein,Ja:Remove tmp files:
-/usr/local/bin/start_dvd2dvd.sh:CLEAN1:0:B:1:Nein,Ja:Clean dirs:
-/usr/local/bin/start_dvd2dvd.sh:VERBOSE1:1:B:1:Nein,Ja:Turns verbosity on:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:ACTION1:0:L:1:all,copy,demux,encode,reqant,mplex,dvdauthor,iso,move:Actions:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:LANGUAGE1:0:L:1:de,en:Language:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:OSDINFO1:1:B:0:Nein,Ja:OSD-Info:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:TITLE1: :A:15:abcdefghijklmnopqrstuvwxyz0123456789-_ABCDEFGHIJKLMNOPQRSTUVWXYZ:Title (optional):
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:TITLENUM1: :A:3:0,99,MAX,AUTO:Specific title (00-99) or (MAX = max. Frames) or (AUTO = vobcopy-test)(optional):
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:ACTION_DEMUX1:0:L:0:tcextract,projectX:Demuxer:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:WITHOUT_X1:0:B:1:Nein,Ja:Start Xvfb (optional):
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:A52DEC_GAIN1: :I:2:-96,96:Audio gain (-96.0 to +96.0)(optional):
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:ACTION_ENCODE1:0:L:0:toolame,mp2enc:Encoder:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:ACTION_MPLEX1:0:L:0:tcmplex-panteltje:Remuxer:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:BURN1:1:B:1:Nein,Ja:Burn DVD:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:RW_FORMAT1:0:B:1:Nein,Ja:Format DVD-RW:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:REMOVE_ISO1:0:B:1:Nein,Ja:Remove this iso file:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:REMOVE1:0:B:1:Nein,Ja:Remove tmp files:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:CLEAN1:0:B:1:Nein,Ja:Clean dirs:
+/usr/share/vdr/dvdconvert/bin/start_dvd2dvd.sh:VERBOSE1:1:B:1:Nein,Ja:Turns verbosity on:
diff -Naur dvdconvert-1.0.2.orig/scripts/dvdconvert.sh dvdconvert-1.0.2/scripts/dvdconvert.sh
--- dvdconvert-1.0.2.orig/scripts/dvdconvert.sh	2007-01-05 20:09:55.284635374 +0100
+++ dvdconvert-1.0.2/scripts/dvdconvert.sh	2007-01-05 20:10:06.545277082 +0100
@@ -1,9 +1,9 @@
 #!/bin/sh
-CFG_FILE="/etc/vdr/plugins/dvdconvert.conf"
-CFG_SAVE="/etc/vdr/plugins/dvdconvert.conf.save"
+CFG_FILE="/usr/share/vdr/dvdconvert/dvdconvert.conf"
+CFG_SAVE="/usr/share/vdr/dvdconvert/dvdconvert.conf.save"
 
 TODO_FILE="/tmp/~dvdconvert.sh"
-LOG_FILE="/var/log/dvdconvert.log"
+LOG_FILE="/var/log/dvdconvert/dvdconvert.log"
 rm -rf $TODO_FILE > /dev/null 2>&1
 echo "#!/bin/sh" > $TODO_FILE
 if [ -s $CFG_SAVE ] ; then
diff -Naur dvdconvert-1.0.2.orig/scripts/getadmval.sh dvdconvert-1.0.2/scripts/getadmval.sh
--- dvdconvert-1.0.2.orig/scripts/getadmval.sh	2007-01-05 20:09:55.284635374 +0100
+++ dvdconvert-1.0.2/scripts/getadmval.sh	2007-01-05 20:10:06.553277538 +0100
@@ -1,5 +1,5 @@
 #!/bin/sh
-CFG_FILE="/etc/vdr/plugins/dvdconvert.conf"
+CFG_FILE="/usr/share/vdr/dvdconvert/dvdconvert.conf"
 
 VAL=`cat $CFG_FILE | grep ":$1:" | cut -f 3 -d ":"`
 TYPE=`cat $CFG_FILE | grep ":$1:" | cut -f 4 -d ":"`
diff -Naur dvdconvert-1.0.2.orig/scripts/setadmval.sh dvdconvert-1.0.2/scripts/setadmval.sh
--- dvdconvert-1.0.2.orig/scripts/setadmval.sh	2007-01-05 20:09:55.284635374 +0100
+++ dvdconvert-1.0.2/scripts/setadmval.sh	2007-01-05 20:10:06.561277993 +0100
@@ -1,5 +1,5 @@
 #!/bin/sh
-CFG_FILE="/etc/vdr/plugins/dvdconvert.conf"
+CFG_FILE="/usr/share/vdr/dvdconvert/dvdconvert.conf"
 
 VAL=$2
 NAME=$1
diff -Naur dvdconvert-1.0.2.orig/scripts/start_dvd2dvd.sh dvdconvert-1.0.2/scripts/start_dvd2dvd.sh
--- dvdconvert-1.0.2.orig/scripts/start_dvd2dvd.sh	2007-01-05 20:09:55.292635831 +0100
+++ dvdconvert-1.0.2/scripts/start_dvd2dvd.sh	2007-01-05 20:10:06.569278450 +0100
@@ -1,10 +1,10 @@
 #!/bin/bash
 
-GETVAL="sh /usr/local/bin/getadmval.sh"
-SETVAL="sh /usr/local/bin/setadmval.sh"
-DVD2DVD_CFG="/etc/vdr/plugins/dvd2dvd.conf"
-DVD2DVD="/usr/local/bin/dvd2dvd.sh"
-LOG_FILE="/var/log/vdr/dvd2dvd.log"
+GETVAL="sh /usr/share/vdr/dvdconvert/bin/getadmval.sh"
+SETVAL="sh /usr/share/vdr/dvdconvert/bin/setadmval.sh"
+DVD2DVD_CFG="/etc/vdr/plugins/dvdconvert/dvdconvert"
+DVD2DVD="/usr/share/vdr/dvdconvert/bin/dvd2dvd.sh"
+LOG_FILE="/var/log/dvdconvert/dvd2dvd.log"
 
 ACTION=`$GETVAL ACTION1`
 echo "ACTION="$ACTION > $DVD2DVD_CFG
diff -Naur dvdconvert-1.0.2.orig/scripts/start_dvd2vdr.sh dvdconvert-1.0.2/scripts/start_dvd2vdr.sh
--- dvdconvert-1.0.2.orig/scripts/start_dvd2vdr.sh	2007-01-05 20:09:55.276634919 +0100
+++ dvdconvert-1.0.2/scripts/start_dvd2vdr.sh	2007-01-05 20:10:06.577278905 +0100
@@ -1,10 +1,10 @@
 #!/bin/bash
 
-GETVAL="sh /usr/local/bin/getadmval.sh"
-SETVAL="sh /usr/local/bin/setadmval.sh"
-DVD2VDR_CFG="/etc/vdr/plugins/dvd2vdr.conf"
-DVD2VDR="/usr/local/bin/dvd2vdr.sh"
-LOG_FILE="/var/log/vdr/dvd2vdr.log"
+GETVAL="sh /usr/share/vdr/dvdconvert/bin/getadmval.sh"
+SETVAL="sh /usr/share/vdr/dvdconvert/bin/setadmval.sh"
+DVD2VDR_CFG="/etc/vdr/plugins/dvdconvert/dvdconvert"
+DVD2VDR="/usr/share/vdr/dvdconvert/bin/dvd2vdr.sh"
+LOG_FILE="/var/log/dvdconvert/dvd2vdr.log"
 
 ACTION=`$GETVAL ACTION`
 echo "ACTION="$ACTION > $DVD2VDR_CFG
